pkgs/by-name/fa/fastcap/package.nix

```
@@ -0,0 +1,88 @@
+{ stdenv
+, fetchzip
+, ghostscript
+, lib
+, texlive
+}:
+
+stdenv.mkDerivation rec {
+  pname = "fastcap";
+  version = "2.0-18Sep92";
+
+  src = fetchzip {
+    url = "http://www.rle.mit.edu/cpg/codes/fastcap-${version}.tgz";
+    hash = "sha256-fnmC6WNd7xk8fphxkMZUq2+Qz+2mWIP2lvBUBAmUvHI";
+    stripRoot=false;
+  };
+
+  patches = [
+    ./patches/fastcap-mulGlobal-drop_conflicting_lib.patch
+    ./patches/fastcap-mulSetup-add_forward_declarations.patch
```

Files and dirs in general should be named lowercase with dashes: https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#file-naming-and-organisation

```suggestion
    ./patches/fastcap-mulglobal-drop-conflicting-lib.patch
    ./patches/fastcap-mulsetup-add-forward-declarations.patch
```

---

OK, I'll do it.